integration-libs/cdc/core/auth/services/user-authentication/cdc-login.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from "@angular/core";
+import { LoginFormComponentService } from "feature-libs/user/account/components/login-form";
+import { CdcJsService } from "integration-libs/cdc/root";
+import { AuthService } from "projects/core/src/auth";
```

Comment:
Change import of AuthService, GlobalMessageService, WindowRef to @spartacus/core